devimint/src/main.rs

```diff
@@ -592,6 +592,78 @@ async fn cli_tests(dev_fed: DevFed) -> Result<()> {
     Ok(())
 }
 
+async fn lightning_gw_reconnect_test(dev_fed: DevFed, process_mgr: &ProcessManager) -> Result<()> {
+    #[allow(unused_variables)]
+    let DevFed {
+        bitcoind,
+        cln,
+        lnd,
+        fed,
+        gw_cln,
+        gw_lnd,
+        electrs,
+        esplora,
+    } = dev_fed;
+
+    // Drop other references to CLN and LND so that the test can kill them
+    drop(cln);
+    drop(lnd);
+
+    let gateways = vec![gw_cln, gw_lnd];
+
+    for mut gw in gateways {
+        // Verify that the gateway can query the lightning node for the pubkey and alias
+        let mut cmd = cmd!(gw, "info");
```

Comment:
if you are reusing this `cmd` then give it a good name say `info_cmd`. But repeating `cmd!(gw, "info")` everytime is also not bad.